blog/_posts/2016-03-27-arrays-iteration.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
 title:  Generalizing AbstractArrays: opportunities and challenges
```

Comment:
Removing the web link didn't seem to fix it build issue - I think it's the unescaped second colon in `title: Generalizing AbstractArrays: opportunities and challenges` that's causing the trouble. Quoting the title fixed it for me, e.g. `title:  "Generalizing AbstractArrays: opportunities and challenges"`.